plugins/rest-plugin/src/test/resources/templates/Client.json

```diff
@@ -1,5 +1,5 @@
 {
-  "id": %id,
-  "email": "%email",
-  "name": "%name"
+  "id": {id},
```

Comment:
let's make valid json in templates.
$var and {var} are not valid entities at all.
Let's talk how to do it more strict.